test/plot-schema.json

```diff
@@ -1463,7 +1463,8 @@
       "values": [
        "auto",
        "linear",
-       "array"
+       "array",
+       "sync"
```

Comment:
`colorbar.tickmode` should not have sync option.
You can override `tickmode` for `colorbar` here:
https://github.com/plotly/plotly.js/blob/4db333c44357c36327b966217c6f831c775d01fa/src/components/colorbar/attributes.js#L132

Something like:
```js
    tickmode: extendFlat({}, axesAttrs.tickmode, {
      dflt: values: ['auto', 'linear', 'array'],
      description: { /* only include available options */ }
    }),
```

Reply:
Similarly for `indicator` trace we don't need this option. 
Please modify here:
https://github.com/plotly/plotly.js/blob/4db333c44357c36327b966217c6f831c775d01fa/src/traces/indicator/attributes.js#L310